config.json

```diff
@@ -835,6 +835,16 @@
       ]
     },
     {
+      "uuid": "8c89f739-05fb-7b80-b5f9-6ad079c750ba8302be8",
+      "slug": "two-bucket",
+      "difficulty": 5,
+      "topics": [
+        "algorithms",
```

Comment:
We've switched to `snake_case` naming convention for topics to increase consistency among tracks, so could you please choose them from the following list - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt

Reply:
Sure 👍